### ISSUE TITLE
require on lowdb npm module fails with version 0.15.x of lowdb npm module

### ISSUE THREAD

**Issue by rprattipati** [typicode/lowdb#160]
package.json shows the following where as there is no lib directory in the published npm module. 
![screen shot 2017-02-08 at 2 48 55 pm](https://cloud.githubusercontent.com/assets/3281398/22760990/be16167c-ee0d-11e6-9980-230e38928a44.png)

  "main": "./lib/index.node.js",

**Comment by srikantkt** [comment]
Basically this breaks
`const low = require('lowdb');`

if package.json  dependency is listed as 
```
  "dependencies": {
   "lowdb": "0.15.x"
  }
```

where as it works if it is 
```
  "dependencies": {
   "lowdb": "0.14.x"
  }
```

**Comment by typicode** [comment]
Sorry for the bad release and thank you for the heads up!
I've just fixed it in `v0.15.3`.

**Comment by rprattipati** [comment]
I tested successfully with lowdb@0.15.4. Thanks for the prompt resolution.
